src/psyclone/dynamo0p3_builtins.py

```diff
@@ -737,16 +766,16 @@ def gen_code(self, parent):
 
 
 # ------------------------------------------------------------------- #
-# ============== Scaling fields ===================================== #
+# ============== Scaling real fields ================================ #
 # ------------------------------------------------------------------- #
 
 
 class DynATimesXKern(DynBuiltIn):
-    ''' Multiply the first field by a scalar and return the result as
-    a second field (Y = a.X) '''
+    ''' Multiply the first real-valued field by a real scalar and return
```

Comment:
Commas, full stop and blank line please.

Reply:
Yep.